### ISSUE TITLE
Add the possibility to upload PDFs in the front part of the application to read it

### ISSUE THREAD

**Issue by maxime-dlabai** [serge-chat/serge#17]
Create on the front end the possibility to upload a PDF file for example with the following procedure: 

- Parsing the file to read it
- Split content to respect "limit" of tokens for model to send.
- To be able to make queries on the elements of the document from the prompt.

**Comment by maximeseth** [comment]
(Of course it's not critical but it would really be a plus on the front)

**Comment by nsarrazin** [comment]
Yeah that would be super useful agreed! I think we need to implement a LangChain integration (see #4) in order to do this the easy way and not reinvent the wheel. 

I will try to work on this in the near future

**Comment by maximeseth** [comment]
I'll look into the LangChain integration on my end, if I can help.

**Comment by nsarrazin** [comment]
For sure, feel free to share your findings
